core/irCheck.ml

```diff
@@ -547,14 +593,19 @@ struct
               end
             else
               begin
-              ensure (eq_types (`Value orig) (o#extract_type_equality_context ()) (vt, t) || is_sub_type (vt, t)) (Printf.sprintf "coercion error: %s is not a subtype of %s"
-                                         (string_of_datatype vt) (string_of_datatype t)) (`Value orig);
+              ensure (eq_types (SVal orig) (o#extract_type_equality_context ()) (vt, t) || is_sub_type (vt, t)) (Printf.sprintf "coercion error: %s is not a subtype of %s"
+                                         (string_of_datatype vt) (string_of_datatype t)) (SVal orig);
               Coerce (v, t), t, o
               end
 
+      in
+      fun value -> translate_helper_exns value_inner value (SVal value)
+
```

Comment:
Why so many empty lines?

Reply:
Many of the larger methods in `irCheck` now start with a big nested function, which is used at the end. I tried to make that visually more obvious, but I guess it's a matter of taste.

Reply:
Hm... I don't feel having 4-5 empty lines looks good or helpful. Wonder what others think.

Reply:
I just tend to keep scrolling past the end of very long functions/methods when the next one starts immediately afterwards. But I'm happy to remove those extra lines.